### PR TITLE
RHDEVDOCS-4081 - 5.4.4, 5.3.10, 5.2.13 Z-Stream RNs

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.8+

Issue: https://issues.redhat.com/browse/RHDEVDOCS-4081

Link to docs preview: https://drive.google.com/file/d/1JqPbFD9Jb9NMyQ83_0r0nbGhYgQXr81Q/view?usp=sharing

Reviewers:
@jcantrill / @alanconway 
@kabirbhartiRH / @QiaolingTang / @anpingli

### CODE REVIEW

**Review by jcantrill — COMMENTED** [review]
/lgtm

File: modules/cluster-logging-rn-5.4.4.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+[id="cluster-logging-release-notes-5-4-4"]
+= Logging 5.4.4
+This release includes link:https://access.redhat.com/errata/RHBA-2022:5907[RHBA-2022:5907-OpenShift Logging Bug Fix Release 5.4.4].
+
+[id="openshift-logging-5-4-4-bug-fixes"]
+== Bug fixes
+
+* Before this update, non-latin characters displayed incorrectly in Kibana. With this update, Kibana displays all valid UTF-8 symbols correctly. (link:https://issues.redhat.com/browse/LOG-2794[LOG-2794])
+
+* Before this update, the `cluster-logging` Operator expected a secret to be automatically created by the cluster. Starting with {product-title} 4.11, this behavior changed which caused logging deployments to fail. With this update, the `cluster-logging` Operator resolves the issue by creating the secret if needed.(link:https://issues.redhat.com/browse/LOG-2840[LOG-2840])
```
Comment (rolfedh) [inline modules/cluster-logging-rn-5.4.4.adoc:12]:
```suggestion
* Before this update, the `cluster-logging` Operator relied on the cluster to create a secret. This cluster behavior changed in {product-title} 4.11, which caused logging deployments to fail. With this update, the `cluster-logging` Operator resolves the issue by creating the secret if needed.
(link:https://issues.redhat.com/browse/LOG-2840[LOG-2840])
```

File: modules/cluster-logging-rn-5.4.4.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+[id="openshift-logging-5-4-4-CVEs"]
```
Comment (rolfedh) [inline modules/cluster-logging-rn-5.4.4.adoc:14]: lower case

**Review by rolfedh — APPROVED** [review]

File: modules/cluster-logging-rn-5.3.10.adoc [hunk 3]
```diff
@@ -0,0 +1,31 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-3-10"]
+= Logging 5.3.10
+This release includes link:https://access.redhat.com/errata/RHSA-2022:5908[RHSA-2022:5908-OpenShift Logging Bug Fix Release 5.3.10].
+
+[id="openshift-logging-5-3-10-bug-fixes"]
+== Bug fixes
+* https://bugzilla.redhat.com/show_bug.cgi?id=2100495[BZ-2100495]
+
```
Comment (kabirbhartiRH) [inline modules/cluster-logging-rn-5.3.10.adoc:10]: There is another JIRA for 5.3.10 [LOG-2817](https://issues.redhat.com/browse/LOG-2817) I don't see no-rn label on this one.

File: modules/cluster-logging-rn-5.4.4.adoc [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+= Logging 5.4.4
+This release includes link:https://access.redhat.com/errata/RHBA-2022:5907[RHBA-2022:5907-OpenShift Logging Bug Fix Release 5.4.4].
+
+[id="openshift-logging-5-4-4-bug-fixes"]
+== Bug fixes
+
+* Before this update, non-latin characters displayed incorrectly in Kibana. With this update, Kibana displays all valid UTF-8 symbols correctly. (link:https://issues.redhat.com/browse/LOG-2794[LOG-2794])
+
+* Before this update, the `cluster-logging` Operator relied on the cluster to create a secret. This cluster behavior changed in {product-title} 4.11, which caused logging deployments to fail. With this update, the `cluster-logging` Operator resolves the issue by creating the secret if needed. (link:https://issues.redhat.com/browse/LOG-2840[LOG-2840])
+
```
Comment (kabirbhartiRH) [inline modules/cluster-logging-rn-5.4.4.adoc:13]: @libander Could you check if the below bugs for 5.4.4 need no-rn label? They are in errata but not in docs.
https://issues.redhat.com/browse/LOG-2657 
https://issues.redhat.com/browse/LOG-2821

File: modules/cluster-logging-rn-5.3.10.adoc (hunk 3 above)
Comment (kabirbhartiRH) [reply modules/cluster-logging-rn-5.3.10.adoc:10]: Ignore this comment! This was a CVE :)

Comment (kabirbhartiRH) [reply modules/cluster-logging-rn-5.3.10.adoc:10]: > There is another JIRA for 5.3.10 [LOG-2817](https://issues.redhat.com/browse/LOG-2817) I don't see _no-rn_ label on this one.

Ignore this comment! This was a CVE :)

Comment (libander) [reply modules/cluster-logging-rn-5.3.10.adoc:10]: [LOG-2817](https://issues.redhat.com/browse/LOG-2817) Is the private JIRA issue for https://bugzilla.redhat.com/show_bug.cgi?id=2100495[BZ-2100495]

**Pull request merged**